driver/sccb.c

```diff
@@ -77,6 +103,7 @@ uint8_t SCCB_Probe()
 
 uint8_t SCCB_Read(uint8_t slv_addr, uint8_t reg)
 {
+	ESP_LOGD(TAG,"SCCB_Read: addr=0x%X, reg=0x%X",slv_addr,reg);
```

Comment:
maybe the more proper log level here would be `Verbose`. Debug is often used otherwise and would make the driver a bit too chatty :)